LEAF_Request_Portal/admin/index.php

```diff
@@ -61,9 +61,16 @@
 
 $action = isset($_GET['a']) ? XSSHelpers::xscrub($_GET['a']) : '';
 
-function customTemplate($tpl)
+function customTemplate($tpl, $config)
```

Comment:
Since all you're using from the config is portalPath, it would be better to just pass in portalPath, instead of taking the whole config.

Reply:
done